src/components/Thanks/BorrowerAvatarsContainer.vue

```diff
@@ -92,6 +95,31 @@ const getLoanImageUrl = loan => {
 	});
 };
 
+const setRef = el => {
+	if (el) avatarsRefs.value.push(el);
+};
+
+const waitForImageToComplete = img => {
+	return new Promise(resolve => {
+		const check = () => {
+			if (img?.complete) {
+				resolve(img);
+			} else {
+				setTimeout(check, 100);
+			}
+		};
+		check();
+	});
+};
+
+onMounted(() => {
+	avatarsRefs.value.forEach(async el => {
+		const img = el.imageRef;
+		await waitForImageToComplete(img);
```

Comment:
Is this something we can do within the avatar component? Otherwise we'll have to do this change with each use case.